sway/input/cursor.c

```diff
@@ -44,7 +44,8 @@ static struct wlr_surface *layer_surface_at(struct sway_output *output,
  * Returns the container at the cursor's position. If there is a surface at that
  * location, it is stored in **surface (it may not be a view).
  */
-static struct sway_container *container_at_cursor(struct sway_cursor *cursor,
+static struct sway_container *container_at_coords(
```

Comment:
We already have `container_at`, adding `container_at_coords` is confusing.

The doc comment should be updated.

Reply:
This function does too much, we should split it into container_at and surface_at and update a bunch of stuff, but that's out of scope for this PR.